### DISCUSSION TITLE
Is there a particular reason Welch method is the default PSD estimation method?

### DISCUSSION THREAD

**Discussion by himalayajung** [neuropsychology/NeuroKit#876] [Q&A]
Is there a particular reason Welch method is the default PSD estimation method in https://github.com/neuropsychology/NeuroKit/blob/master/neurokit2/hrv/hrv_frequency.py. What is the current consensus on the 'best' PSD estimation method?

Related question https://github.com/Tam-Pham/HRVStructure/issues/3

**Comment by welcome** [comment]
Hi 👋 Thanks for reaching out and opening your first issue here! We'll try to come back to you as soon as possible. ❤️ ![kenobi](https://media.giphy.com/media/Nx0rz3jtxtEre/giphy.gif)
